### PR TITLE
Release notes 118 OSSM

### PR DESCRIPTION
**Pull request by neal-timpe**
https://issues.redhat.com/browse/OSSMDOC-121

### CODE REVIEW

File: modules/ossm-rn-new-features.adoc [hunk 1]
```diff
@@ -23,18 +23,22 @@ Result – If changed, describe the current user experience
...
-|1.12.7
+|1.12.15
 
 |3scale Istio Adapter
 |1.0.0
 |===
 
+== New features {ProductName} 1.1.8
+
+This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes. This release upgrades Istio to version 1.4.10.
```
Comment (longmuir) [inline modules/ossm-rn-new-features.adoc:40]: Minor thing, but perhaps link to the Istio 1.4.10 release notes: https://istio.io/latest/news/releases/1.4.x/announcing-1.4.10/ >

**Review by tvieira — APPROVED** [review]
LGTM

**Comment by neal-timpe** [comment]
@yxun

**Comment by yxun** [comment]
LGTM

**Review by gbaufake — APPROVED** [review]
LGTM with issues fixed

**Review by JStickler — APPROVED** [review]

**Pull request merged**